src/components/org/apache/jmeter/assertions/BSFAssertion.java

```diff
@@ -57,8 +57,4 @@ public AssertionResult getResult(SampleResult response) {
         return result;
     }
     
-    @Override
-    public Object clone() {
```

Comment:
If we remove this method sonar will complain about the missing method, as the class is marked Clonable